bottles/frontend/views/bottle_preferences.py

```diff
@@ -198,7 +202,19 @@ def __init__(self, details, config, **kwargs):
         self.entry_name.connect("apply", self.__save_name)
         # endregion
 
-        if RuntimeManager.get_runtimes("bottles"):
+        """Set DXVK_NVAPI related rows to visible when an NVIDIA GPU is detected (invisible by default)"""
+        try:
```

Comment:
use `contextlib` to mute an exception:

```python
import contextlib # on top of the document ofc

with contextlib.suppress(KeyError):
    vendor = gpu["vendors"]["nvidia"]["vendor"]
    if vendor == "nvidia":
        self.row_nvapi.set_visible(True)
        self.row_nvapi_version.set_visible(True)
```

I've also removed the raised `KeyError` as we are already in a condition